saltychat/SaltyClient/VoiceManager.cs

```diff
@@ -489,6 +494,22 @@ internal void SetRadioChannel(string radioChannelName, bool primary)
 
             BaseScript.TriggerServerEvent(Event.SaltyChat_SetRadioChannel, radioChannelName, primary);
         }
+
+        internal void SetRadioVolume(float radioVolumeNewValue)
```

Comment:
Parameter was named `volumeLevel` in the readme, but now it's `radioVolumeNewValue`?
```suggestion
        internal void SetRadioVolume(float volumeLevel)
```